app/models/miq_policy_set.rb

```diff
@@ -116,11 +116,44 @@ def self.import_from_yaml(fd)
   end
 
   def self.seed
+    seed_fixtures
     all.each do |ps|
       if ps.mode.nil?
         _log.info("Updating [#{ps.name}]")
         ps.update_attribute(:mode, "control")
       end
     end
   end
+
+  def self.seed_fixtures
+    fixture_file = File.join(FIXTURE_DIR, "miq_policy_sets.yml")
+    fixtures = File.exist?(fixture_file) ? YAML.load_file(fixture_file) : []
+
+    fixtures.each do |policy_set_fixture|
+      rec = seed_policy_sets(policy_set_fixture.except(:miq_policies))
```

Comment:
Same as ☝️, singular... since the method only creates/finds a single policy set.